### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/standard/analyzers/portability-analyzer.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ Want to make your libraries multi-platform? Want to see how much work is require
 
 ## New targets
 
-* [.NET Core](https://dotnetfoundation.org/net-core): Has a modular design, employs side-by-side, and targets cross-platform scenarios. Side-by-side allows you to adopt new .NET Core versions without breaking other apps.
-* [ASP.NET Core](https://dotnetfoundation.org/asp-net-core): is a modern web-framework built on .NET Core thus giving developers the same benefits.
+* [.NET Core](../../core/index.md): Has a modular design, employs side-by-side, and targets cross-platform scenarios. Side-by-side allows you to adopt new .NET Core versions without breaking other apps.
+* [ASP.NET Core](/aspnet/core): is a modern web-framework built on .NET Core thus giving developers the same benefits.
```
Comment (JRAlexander) [inline docs/standard/analyzers/portability-analyzer.md]: Remove the colon.

File: docs/standard/analyzers/portability-analyzer.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ Want to make your libraries multi-platform? Want to see how much work is require
...
 * [Universal Windows Platform](https://blogs.msdn.microsoft.com/dotnet/2014/04/24/net-native-performance): Improve performance of your Windows Store apps that run on x64 and ARM machines by using .NET Native’s static compilation. 
 * .NET Core + Platform Extensions: Includes the .NET Core APIs in addition to other APIs in the .NET ecosystem such as WCF, ASP.NET Core, FSharp, and Azure.
```
Comment (JRAlexander) [inline docs/standard/analyzers/portability-analyzer.md]: Should there be a link for this as well?

File: docs/standard/analyzers/portability-analyzer.md [hunk 3]
```diff
@@ -21,8 +21,8 @@ Want to make your libraries multi-platform? Want to see how much work is require
...
 * .NET Standard + Platform Extensions: Includes the .NET Standard APIs in addition to other .NET ecosystem such as WCF, ASP.NET Core, FSharp, and Azure.
```
Comment (JRAlexander) [inline docs/standard/analyzers/portability-analyzer.md]: And for this one?

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Looks good - Just a few small items.

File: docs/standard/analyzers/portability-analyzer.md (hunk 1 above)
Comment (mairaw) [reply docs/standard/analyzers/portability-analyzer.md]: so should I remove from all entries? They all have colons.

File: docs/standard/analyzers/portability-analyzer.md (hunk 2 above)
Comment (mairaw) [reply docs/standard/analyzers/portability-analyzer.md]: Nope. We don't have anywhere to link to where we talk about this.

File: docs/standard/analyzers/portability-analyzer.md (hunk 3 above)
Comment (mairaw) [reply docs/standard/analyzers/portability-analyzer.md]: Same here.

**Comment by mairaw** [comment]
@JRAlexander please let me know. The changes I made are mostly intended to fix some of the current broken links.

File: docs/standard/analyzers/portability-analyzer.md (hunk 1 above)
Comment (JRAlexander) [reply docs/standard/analyzers/portability-analyzer.md]: Nope

File: docs/standard/analyzers/portability-analyzer.md (hunk 2 above)
Comment (JRAlexander) [reply docs/standard/analyzers/portability-analyzer.md]: ok

File: docs/standard/analyzers/portability-analyzer.md (hunk 3 above)
Comment (JRAlexander) [reply docs/standard/analyzers/portability-analyzer.md]: ok

**Review by JRAlexander — APPROVED** [review]

**Pull request merged**